### PR TITLE
[SW-2106] Remove and replace remaining reference in to H2O algos in SW doc

### PR DESCRIPTION
**Pull request by jakubhava**

### CODE REVIEW

File: doc/src/site/sphinx/tutorials/spark_h2o_conversions.rst [hunk 1]
```diff
@@ -11,108 +11,85 @@ Quick links:
...
-
---------------
+    val h2oFrame = h2oContext.asH2OFrame(rdd)
 
 Converting a DataFrame into an H2OFrame
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-The ``H2OContext`` provides **implicit** conversion from the specified ``DataFrame`` to ``H2OFrame``. The conversion will create a new ``H2OFrame``, transfer data from the specified ``DataFrame``, and save it to the H2O K/V data store.
+The ``H2OContext`` provides conversion method from the specified ``DataFrame`` to ``H2OFrame``.
+The conversion creates a new ``H2OFrame``, transfer data from the specified ``DataFrame``, and saves it to the H2O
```
Comment (mn-mikke) [inline doc/src/site/sphinx/tutorials/spark_h2o_conversions.rst]: `transfer` -> `transfers`

File: doc/src/site/sphinx/tutorials/spark_h2o_conversions.rst [hunk 2]
```diff
@@ -11,108 +11,85 @@ Quick links:
...
---------------
+    val dataFrame = asDataFrame(h2oFrame)
 
 Converting an RDD[T] into an H2OFrame
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-The ``H2OContext`` provides **implicit** conversion from the specified ``RDD[A]`` to ``H2OFrame``. As with conversion in the opposite direction, the type ``A`` has to satisfy the upper bound expressed by the type ``Product``. The conversion will create a new ``H2OFrame``, transfer data from the specified RDD, and save it to the H2O K/V data store.
+The ``H2OContext`` provides conversion method from the specified ``RDD[A]`` to ``H2OFrame``. As with conversion
+in the opposite direction, the type ``A`` has to satisfy the upper bound expressed by the type ``Product``. The conversion
+creates a new ``H2OFrame``, transfers data from the specified RDD, and saves it to to the H2O backend.
```
Comment (mn-mikke) [inline doc/src/site/sphinx/tutorials/spark_h2o_conversions.rst]: Couldn't `H2O backend` be misinterpreted with the external/internal backend?

File: py/src/ai/h2o/sparkling/ml/params/H2ODeepLearningParams.py [hunk 3]
```diff
@@ -61,6 +60,12 @@ class H2ODeepLearningParams(H2OAlgoSupervisedParams, HasStoppingCriteria, HasQua
         "Force reproducibility on small data (will be slow - only uses 1 thread)",
         H2OTypeConverters.toBoolean())
 
+    activation = Param(
+        Params._dummy(),
+        "activation",
+        "The activation function (non-linearity) to be used the neurons in the hidden layers.",
```
Comment (mn-mikke) [inline py/src/ai/h2o/sparkling/ml/params/H2ODeepLearningParams.py]: * -> `The activation function (non-linearity) applied on neurons of hidden layers.`?

File: ml/src/main/scala/ai/h2o/sparkling/ml/params/H2ODeepLearningParams.scala [hunk 4]
```diff
@@ -37,6 +39,8 @@ trait H2ODeepLearningParams extends H2OAlgoSupervisedParams[DeepLearningParamete
   private val l2 = doubleParam("l2")
   private val hidden = intArrayParam("hidden")
   private val reproducible = booleanParam("reproducible")
+  private val activation = stringParam("activation")
```
Comment (mn-mikke) [inline ml/src/main/scala/ai/h2o/sparkling/ml/params/H2ODeepLearningParams.scala]: Is there any enum on H2O side for activation functions?

Comment (jakubhava) [reply ml/src/main/scala/ai/h2o/sparkling/ml/params/H2ODeepLearningParams.scala]: yes, we are using it :) But on scala side we represent enum values as string

Comment (mn-mikke) [reply ml/src/main/scala/ai/h2o/sparkling/ml/params/H2ODeepLearningParams.scala]: ok, thanks!

File: doc/src/site/sphinx/tutorials/spark_h2o_conversions.rst (hunk 1 above)
Comment (jakubhava) [reply doc/src/site/sphinx/tutorials/spark_h2o_conversions.rst]: thx!

File: doc/src/site/sphinx/tutorials/spark_h2o_conversions.rst (hunk 2 above)
Comment (jakubhava) [reply doc/src/site/sphinx/tutorials/spark_h2o_conversions.rst]: Good catch -> reworded to `to DKV store on the H2O backend.`, WDYT?

Comment (mn-mikke) [reply doc/src/site/sphinx/tutorials/spark_h2o_conversions.rst]: Sounds good to me

**Pull request merged**